internal/sbi/api_parameterprovision.go

```diff
@@ -23,9 +23,86 @@ func (s *Server) getParameterProvisionRoutes() []Route {
 		{
 			"Update",
 			strings.ToUpper("Patch"),
-			"/:gpsi/pp-data",
+			"/:ueId/pp-data",
 			s.HandleUpdate,
 		},
+
+		{
+			"Create5GMBSGroup",
+			strings.ToUpper("Put"),
+			"/mbs-group-membership/:extGroupId",
+			s.HandleCreate5GMBSGroup,
+		},
+
+		{
+			"Create5GVNGroup",
+			strings.ToUpper("Put"),
+			"/5g-vn-groups/:extGroupId",
+			s.HandleCreate5GVNGroup,
+		},
+
+		{
+			"CreatePPDataEntry",
+			strings.ToUpper("Put"),
+			"/:ueId/pp-data-store/:afInstanceId",
+			s.HandleCreatePPDataEntry,
+		},
+
+		{
+			"Delete5GMBSGroup",
+			strings.ToUpper("Delete"),
+			"/mbs-group-membership/:extGroupId",
+			s.HandleDelete5GMBSGroup,
+		},
+
+		{
+			"Delete5GVNGroup",
+			strings.ToUpper("Delete"),
+			"/5g-vn-groups/:extGroupId",
+			s.HandleDelete5GVNGroup,
+		},
+
+		{
+			"DeletePPDataEntry",
+			strings.ToUpper("Delete"),
+			"/:ueId/pp-data-store/:afInstanceId",
+			s.HandleDeletePPDataEntry,
+		},
+
+		{
+			"Get5GMBSGroup",
+			strings.ToUpper("Get"),
+			"/mbs-group-membership/:extGroupId",
+			s.HandleGet5GMBSGroup,
+		},
+
+		{
+			"Get5GVNGroup",
+			strings.ToUpper("Get"),
+			"/5g-vn-groups/:extGroupId",
+			s.HandleGet5GVNGroup,
+		},
+
+		{
+			"GetPPDataEntry",
+			strings.ToUpper("Get"),
+			"/:ueId/pp-data-store/:afInstanceId",
+			s.HandleGetPPDataEntry,
+		},
+
+		{
+			"Modify5GMBSGroup",
+			strings.ToUpper("Patch"),
+			"/mbs-group-membership/:extGroupId",
+			s.HandleModify5GMBSGroup,
+		},
+
+		{
+			"Modify5GVNGroup",
+			strings.ToUpper("Patch"),
+			"/5g-vn-groups/:extGroupId",
+			s.HandleModify5GVNGroup,
+		},
```

Comment:
Use library constants instead of strings

```suggestion
		{
			"Update",
			http.MethodPatch,
			"/:ueId/pp-data",
			s.HandleUpdate,
		},

		{
			"Create5GMBSGroup",
			http.MethodPut,
			"/mbs-group-membership/:extGroupId",
			s.HandleCreate5GMBSGroup,
		},

		{
			"Create5GVNGroup",
			http.MethodPut,
			"/5g-vn-groups/:extGroupId",
			s.HandleCreate5GVNGroup,
		},

		{
			"CreatePPDataEntry",
			http.MethodPut,
			"/:ueId/pp-data-store/:afInstanceId",
			s.HandleCreatePPDataEntry,
		},

		{
			"Delete5GMBSGroup",
			http.MethodDelete,
			"/mbs-group-membership/:extGroupId",
			s.HandleDelete5GMBSGroup,
		},

		{
			"Delete5GVNGroup",
			http.MethodDelete,
			"/5g-vn-groups/:extGroupId",
			s.HandleDelete5GVNGroup,
		},

		{
			"DeletePPDataEntry",
			http.MethodDelete,
			"/:ueId/pp-data-store/:afInstanceId",
			s.HandleDeletePPDataEntry,
		},

		{
			"Get5GMBSGroup",
			http.MethodGet,
			"/mbs-group-membership/:extGroupId",
			s.HandleGet5GMBSGroup,
		},

		{
			"Get5GVNGroup",
			http.MethodGet,
			"/5g-vn-groups/:extGroupId",
			s.HandleGet5GVNGroup,
		},

		{
			"GetPPDataEntry",
			http.MethodGet,
			"/:ueId/pp-data-store/:afInstanceId",
			s.HandleGetPPDataEntry,
		},

		{
			"Modify5GMBSGroup",
			http.MethodPatch,
			"/mbs-group-membership/:extGroupId",
			s.HandleModify5GMBSGroup,
		},

		{
			"Modify5GVNGroup",
			http.MethodPatch,
			"/5g-vn-groups/:extGroupId",
			s.HandleModify5GVNGroup,
		},
```